### DISCUSSION TITLE
How about creating a complete demo?

### DISCUSSION THREAD

**Discussion by frederikhors** [urql-graphql/urql#1306] [Ideas]
@kitten, @JoviDeCroock How about creating a complete demo with everything urql offers?

 If you feel like doing it in React then I can translate it into Svelte.

We can fix questions like https://github.com/FormidableLabs/urql/issues/201#issuecomment-759541490 and I can improve a lot avoiding headaches every time I need to remember how to do something.

**Comment by JoviDeCroock** [comment]
We have this example that we made a while back https://github.com/JoviDeCroock/threed-web that encompasses a lot
